x/authz/codec/cdc.go

```diff
@@ -6,7 +6,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-var Amino = codec.NewLegacyAmino()
+var (
+	Amino     = codec.NewLegacyAmino()
```

Comment:
We should remove this line from the changelog as well if we add them back:
> * (x/authz,consensus,crisis,slashing) [#15675](https://github.com/cosmos/cosmos-sdk/pull/15675) Remove ModuleCdc as `NewAminoCodec` is depreacted.

The Amino here can go back as private too.

Reply:
this needs to stay public, this was previously public as well.